src/figuresClasses.ts

```diff
@@ -1,11 +1,88 @@
-export interface Figure {}
+export interface Figure {
+  type: 'Triangle' | 'Circle' | 'Rectangle';
+  color: 'red' | 'green' | 'blue';
+  shape: string;
```

Comment:
The `shape` property in the `Figure` interface is defined as a `string`, but it should be a union type like `'triangle' | 'circle' | 'rectangle'` to ensure consistency with the implementing classes.